portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +78,20 @@ fetch('/data').then(response => response.json()).then((myData) => {
     const arrayContainer = document.getElementById('array-container');
     arrayContainer.innerText = myData; });
 }
+
+/*
+ * Fetches the comments array and adds each element to the history 
+ * unordered list on the homepage 
+ */
+ function getComments(){
+     fetch('/data').then(response => response.json()).then((comment) => {
+        const historyEl = document.getElementById('history');
+        console.log(comment + " and " + comment.length);
+        for(i=0; i<comment.length; i++){
+            const liElement = document.createElement('li');
```

Comment:
I might extract this logic out into its own method "addCommentToDom()" or something like that.